release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -143,9 +143,185 @@ In {product-title} 4.12, you can do the following from the *Helm* page:
 [id="ocp-4-12-ibm-z"]
 === IBM Z and LinuxONE
 
+With this release, IBM Z and LinuxONE are now compatible with {product-title} {product-version}. The installation can be performed with z/VM or {op-system-base} KVM. For installation instructions, see the following documentation:
+
+* xref:../installing/installing_ibm_z/installing-ibm-z.adoc#installing-ibm-z[Installing a cluster with z/VM on IBM Z and LinuxONE]
+* xref:../installing/installing_ibm_z/installing-restricted-networks-ibm-z.adoc#installing-restricted-networks-ibm-z[Installing a cluster with z/VM on IBM Z and LinuxONE in a restricted network]
+* xref:../installing/installing_ibm_z/installing-ibm-z-kvm.adoc#installing-ibm-z-kvm[Installing a cluster with {op-system-base} KVM on IBM Z and LinuxONE]
+* xref:../installing/installing_ibm_z/installing-restricted-networks-ibm-z-kvm.adoc#installing-restricted-networks-ibm-z-kvm[Installing a cluster with RHEL KVM on IBM Z and LinuxONE in a restricted network]
+
+[discrete]
+==== Notable enhancements
+
+The following new features are supported on IBM Z and LinuxONE with {product-title} {product-version}:
+
+* Cron jobs
+* Descheduler
+* FIPS cryptography
```

Comment:
```suggestion
//* FIPS cryptography
```
@alishaIBM In todays program call we decided to take FIPS cryptography out and readd through change mgmt after GA see here: https://docs.google.com/document/d/1lgW4fqzzPo4C2zBd81vcKBLtbRH7tDuBN8ODKwW8Idk/edit#heading=h.rq8qakhcqcki

Reply:
Acknowledge